### PR TITLE
Rec prep 20220823

### PR DESCRIPTION
**Pull request by omoerbeek**
### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/secpoll.zone [hunk 1]
```diff
@@ -302,21 +302,24 @@ recursor-4.5.4.security-status                          60 IN TXT "3 Upgrade now
 recursor-4.5.5.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-01.html"
 recursor-4.5.6.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-01.html"
 recursor-4.5.7.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-01.html"
-recursor-4.5.8.security-status                          60 IN TXT "1 OK"
-recursor-4.5.9.security-status                          60 IN TXT "1 OK"
+recursor-4.5.8.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-02.html"
+recursor-4.5.9.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-02.html"
+recursor-4.5.10.security-status                          60 IN TXT "1 OK"
```
Comment (aerique) [inline docs/secpoll.zone:307]: nit (I think unless the parser is really strict): extra indent

File: pdns/recursordist/docs/changelog/4.5.rst [hunk 2]
```diff
@@ -1,6 +1,22 @@
 Changelogs for 4.5.X
 ====================
 
+.. changelog::
+  :version: 4.5.10
+  :released: 23th of August 2022
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.5.rst:6]:
```suggestion
  :released: 23rd of August 2022
```

File: docs/secpoll.zone [hunk 3]
```diff
@@ -302,21 +302,24 @@ recursor-4.5.4.security-status                          60 IN TXT "3 Upgrade now
...
 recursor-4.6.0-rc1.security-status                      60 IN TXT "3 Unsupported pre-release (known vulnerabilities)"
 recursor-4.6.0.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-01.html"
-recursor-4.6.1.security-status                          60 IN TXT "1 OK"
-recursor-4.6.2.security-status                          60 IN TXT "1 OK"
+recursor-4.6.1.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-02.html"
+recursor-4.6.2.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2022-02.html"
+recursor-4.6.3.security-status                          60 IN TXT "1 OK"
 recursor-4.7.0-alpha1.security-status                   60 IN TXT "3 Unsupported pre-release (known vulnerabilities)"
 recursor-4.7.0-beta1.security-status                    60 IN TXT "2 Unsupported pre-release"
 recursor-4.7.0-rc1.security-status                      60 IN TXT "2 Unsupported pre-release"
```
Comment (Habbie) [inline docs/secpoll.zone:319]:
```suggestion
recursor-4.7.0-beta1.security-status                    60 IN TXT "2 Unsupported pre-release (known vulnerabilities)"
recursor-4.7.0-rc1.security-status                      60 IN TXT "2 Unsupported pre-release (known vulnerabilities)"
```

File: docs/secpoll.zone (hunk 1 above)
Comment (Habbie) [inline docs/secpoll.zone:307]:
```suggestion
recursor-4.5.10.security-status                         60 IN TXT "1 OK"
```

**Review by aerique — APPROVED** [review]
LGTM

**Review by Habbie — CHANGES_REQUESTED** [review]

**Pull request merged**